app/models/config.rb

```diff
@@ -408,6 +408,12 @@ def validate_shared_reset_days
       end
     end
 
+    def validate_budget_bar_max
+      if !validate_number
```

Comment:
When I put `nah` in here it lets me, though it seems to evaluate to zero -- I think this is because `validate_number` produces a string if something's up. That said it doesn't break anything on the frontend thankfully